extension-manifest-v3/tests/src/helpers.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import os from 'node:os';
+import crypto from 'node:crypto';
+import decompress from 'decompress';
+
+export const sleep = (ms) => new Promise((r) => setTimeout(() => r(), ms));
+
+export const switchToWindowWithUrl = async (context, url) => {
+  const pages = context.pages();
+  for (const page of pages) {
+    if (page.url() === url) {
+      await page.bringToFront();
+      return page;
+    }
+  }
+  throw new Error(`No window with URL: ${url}`);
+};
+
+export const downloadAddon = async (url) => {
+  if (!url) {
+    console.error('INFO: No extension selected.');
```

Comment:
Arguably, it is better to throw an exception here, since it is a logical error if there is no URL.